.github/ISSUE_TEMPLATE

```diff
@@ -0,0 +1,30 @@
+## **SUBMISSIONS LACKING A RESPONSE TO ONE OF THE FIELDS WILL BE DELETED**
```

Comment:
Why are you using `##`? This will only be emphasized _after_ the issue is created.